### ISSUE TITLE
MAJOR: expert_gating_func header field ignored — always sigmoid, never reads field

### ISSUE THREAD

**Issue by bong-water-water-bong** [1bit-MONSTER/1bit-MONSTER#650]
### Files: src/backend_laguna.cpp:704, src/laguna_generate.cpp:368, include/onebp_format.h:94-98,140

The 1BP header stores `expert_gating_func` with values SIGMOID=0 or SOFTMAX=1. But the inference code always hardcodes sigmoid:

```cpp
for (int e = 0; e < NE; e++) router_logits[e] = sigmoid(router_logits[e]);
```

A grep of the entire source tree confirms `expert_gating_func` is never read anywhere outside the header definition and converter.

### Impact
Softmax-gated MoE models produce wrong routing weights. The sigmoid produces independent per-expert probabilities instead of a proper probability distribution. Expert selection and weighting are incorrect.

### Fix
Read `h.expert_gating_func` in both backends. When SOFTMAX, use softmax instead of sigmoid for router activation.

### Severity: MAJOR

**Comment by bong-water-water-bong** [comment]
Fixed in 068bf60 (branch fix/flm-removal-cleanup).
